src/config/hunspell.rs

```diff
@@ -0,0 +1,144 @@
+//! Hunspell checker configuration.
+
+use super::{SearchDirs, WrappedRegex};
+use std::path::{Path, PathBuf};
+
+use anyhow::{bail, Result};
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Deserialize, Serialize, Debug, Clone)]
+pub struct Quirks {
+    /// A regular expression, whose capture groups will be checked, instead of the initial token.
+    /// Only the first one that matches will be used to split the word.
+    #[serde(default)]
+    pub transform_regex: Vec<WrappedRegex>,
+    /// Allow concatenated words instead of dashed connection.
+    /// Note that this only applies, if one of the suggested replacements has an item that is
+    /// equivalent except for addition dashes (`-`).
+    #[serde(default)]
+    pub allow_concatenation: bool,
+    /// The counterpart of `allow_concatenation`. Accepts words which have repalcement suggestions
```

Comment:
```suggestion
    /// The counterpart of `allow_concatenation`. Accepts words which have replacement suggestions
```

Reply:
I just ran `cargo spellcheck --fix` on the codebase, and it turns out CI should enforce it. There were quite a few.